winspool/winspool.go

```diff
@@ -449,8 +462,6 @@ func (ws *WinSpool) GetPrinters() ([]lib.Printer, error) {
 		printers = append(printers, printer)
 	}
 
-	printers = lib.FilterBlacklistPrinters(printers, ws.printerBlacklist)
```

Comment:
Are there still users of these two functions in lib? Can we delete that code? Or maybe update the code in lib to work in a loop like this?

Reply:
Yes, CUPS uses them here:
https://github.com/google/cloud-print-connector/blob/master/cups/cups.go#L225

it seems much earlier for CUPS. I wasn't planning to touch it but if you feel like it'd be best to do the filtering in the loop in both cases I can add that.